.circleci/config.yml

```
@@ -34,21 +34,21 @@ jobs:
           root: .
           paths:
             - .
-  lint-and-flow:
+  lint-and-typescript:
     <<: *defaults
     steps:
       - attach_workspace:
           at: ~/react-native-testing-library
       - run: |
           yarn lint
-          yarn flow-check
-  typescript:
+          yarn typecheck
```

```suggestion
          yarn typescript-check
```
This needs to match package.json script name.